DistributionPackages/Neos.Presentation/Resources/Private/Fusion/Atom/Icon.fusion

```diff
@@ -14,14 +15,15 @@ prototype(Neos.Presentation:Icon) < prototype(Neos.Fusion:Component) {
     fallbackName.@process.clean = ${String.trim(String.toLowerCase(value))}
 
     inline = null
-
     class = null
+    attributes = Neos.Fusion:DataStructure
 
     renderer = Neos.Fusion:Augmenter {
         data-icon-name = ${props.name}
         data-icon-name.@if.hasName = ${props.name}
 
         class = ${props.class}
+        attributes = ${props.attributes}
```

Comment:
I think this don't work, as it is not a Neos.Fusion:Tag. Shouldn't we use `@apply` here?

Reply:
I used it here and it's working: https://github.com/neos/Neos.NeosIo/pull/570/files#diff-848690a40a1ace0905c7261f04fa62147836e1377c165d75523a454d4322808fR131

I like to add attributes to components, as it get's rid of `Neos.Fusion:Augmenter` is many cases.